content/en/docs/reference/kubernetes-api/extend-resources/device-class-v1beta1.md

```diff
@@ -0,0 +1,629 @@
+---
```

Comment:
These files will be regenerated anyway, after the v1.32 is cut.

Reply:
They had to be added to make this PR mergeable.

I included a TODO in the edited files about moving DeviceClass to make it stand out when this file gets overwritten by a generated version and this TODO doesn't get addressed.